services/wallet/common/const.go

```diff
@@ -13,7 +13,26 @@ type MultiTransactionIDType int64
 const (
 	NoMultiTransactionID = MultiTransactionIDType(0)
 	HexAddressLength     = 42
-	StatusDomain         = "stateofus.eth"
+
+	StatusDomain = ".stateofus.eth"
+	EthDomain    = ".eth"
+
+	EthSymbol  = "ETH"
+	SntSymbol  = "SNT"
+	SttSymbol  = "STT"
+	UsdcSymbol = "USDC"
+	HopSymbol  = "HOP"
+
+	ProcessorTransferName     = "Transfer"
```

Comment:
hmm these don't really sound like a "common" thing, it's a very pathprocessor-specific thing. I'd favour doing a new package like `pathprocessortypes` or something like that if it casuses circular dependency, otherwise the `common` package will just become a dump of every constant used in more than one place.